src/components/CommentList.jsx

```diff
@@ -0,0 +1,27 @@
+import { useParams } from "react-router-dom";
+import { useEffect, useState } from "react";
+import CommentCard from "./CommentCard";
+import { getCommentsByReviewId } from "../utils/utils";
+
+function CommentList() {
+  const { review_id } = useParams();
+  const [commentList, setCommentList] = useState([]);
+
+  useEffect(() => {
+    getCommentsByReviewId(review_id)
+      .then((comment) => {
+        setCommentList(comment)
+      })
+  }, []);
+
+  return (
+    <main>
+      <h4><u>Comments to This Review</u></h4>
+      {commentList.map((comment) => {
+          return <CommentCard key={comment.comment_id} comment={comment} />;
+        })}
+    </main>
+  );
```

Comment:
This works great on reviews that do have comments. However, when you go to a review that has no comments, the page breaks. For example review id 14. Would be great to add a message to the page that says it has no comments if this is the case